blog/views.py

```diff
@@ -1,13 +1,19 @@
 from django.contrib import messages
 from django.shortcuts import get_object_or_404, render, redirect
+from django.views.generic import ListView, DetailView
+from .models import Blog
 
 from .models import Blog
 from .forms import BlogForm
 
 # Create your views here.
 
-def index(request):
-    return render(request, 'blog/index.html')
+#def index(request):
+    #return render(request, 'blog/index.html', {})
+
+class IndexView(ListView):
```

Comment:
nice, although for more complex things I prefer "function based views" (this is ok for now though)